logbook-core/src/test/java/org/zalando/logbook/core/StreamHttpLogWriterTest.java

```diff
@@ -1,6 +1,5 @@
 package org.zalando.logbook.core;
 
-import net.jcip.annotations.NotThreadSafe;
```

Comment:
I think this should not be removed; rather, any place we have

```java
javax.annotation.concurrent.NotThreadSafe;
```

it should be replaced by 
```java
import net.jcip.annotations.NotThreadSafe;
```

`@NotThreadSafe` is merely a documentation annotation for our users.

Reply:
> no updates since 2013

It's a simple set of documentation annotations based on the famous JCIP book. IMO, no update was ever needed as concepts like ThreadSafe, ThreadUnsafe and Immutable have not changed since 😅

Reply:
switched to jcip dependency in https://github.com/zalando/logbook/pull/2193/commits/744bbcbee511eae93e370709def5d1488dc29c62